src/rascal/representations/calculator_spherical_expansion.hh

```diff
@@ -1571,43 +1573,75 @@ namespace rascal {
       expansions_coefficients_gradient.resize();
     }
 
-    /* @TODO(felix,max) use the parity of the spherical harmonics to use half
-     * neighbourlist, i.e. C^{ij}_{nlm} = (-1)^l C^{ji}_{nlm}.
-     */
+    // TODO(felix) think about an option to have "global" species,
```

Comment:
Could this be an issue instead? This way it can have more context, and other can help.

Reply:
I don't think it is need. I will eventually do this task but I still need to think how to do it precisely.

Reply:
This is what we decided in the last meeting this was brought up.